### ISSUE TITLE
Delay effect not functional with lsp-plugins-lv2 1.2.16

### ISSUE THREAD

**Issue by Bundy01** [wwmm/easyeffects#3166]
### EasyEffects Version

7.1.6

### What package are you using?

Arch (easyeffects)

### Distribution

ArchLinux

### Describe the bug

With the installation of the latest package of lsp-plugins-lv2 1.2.16, sound is no longer output after the Delay effect.
To get it to work again, I need to reverse the phase on the delay side. In my case, it's the right-hand side.

The sound comes back but I no longer have any delay, even if I set the delay to 1000ms.

Regards.

### Debug Log

<details><summary>Debug Log</summary><br/>

```
Paste your log here
```
</details>

**Comment by violetmage** [comment]
Duplicate of #3153.

We're waiting on the lsp plug-ins flatpak to be updated with the fix.
